src/components/Landing.js

```diff
@@ -0,0 +1,272 @@
+      height: 'auto',
+    },
+  },
+  title: {
+    color: theme.palette.primary.main,
+  },
+  halfScreenLeft: {
+    width: '50%',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    paddingLeft: '10%',
+    [theme.breakpoints.down(1100)]: {
+      width: '47%',
+      paddingLeft: '7%',
+    },
+    [theme.breakpoints.down('sm')]: {
+      width: '80%',
+      marginTop: theme.spacing(4),
+      paddingLeft: 0,
+      marginBottom: theme.spacing(8),
+    },
+  },
+  halfScreenRight: {
+    width: '55%',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    position: 'absolute',
+    right: 0,
+    [theme.breakpoints.down('sm')]: {
+      position: 'relative',
+      width: '100%',
+    },
+  },
+  subtitle: {
+    marginTop: theme.spacing(2),
+  },
+  buttonStyles: {
+    maxWidth: theme.spacing(30),
+    marginTop: theme.spacing(1.5),
+  },
+  wave: {
+    position: 'absolute',
+    bottom: 0,
+    width: '100%',
+    zIndex: -1,
+    [theme.breakpoints.down('sm')]: {
+      display: 'none',
+    },
+  },
+  waveMobile: {
+    display: 'none',
+    position: 'absolute',
+    width: '100%',
+    zIndex: -1,
+    [theme.breakpoints.down('sm')]: {
+      display: 'block',
+    },
+  },
+}))
+
+const Landing = ({ pageContext, location, causeData }) => {
+  const {
+    canonicalURL,
+    headTitle,
+    headTitleTemplate,
+    headOgTitle,
+    headOgDescription,
+    headKeywords,
+    headOgImgURLAbsolute,
+    title,
+    subtitle,
+    causeId,
+  } = causeData
+  const cx = useStyles()
+  const [showUnsupportedBrowserMessage, setShowUnsupportedBrowserMessage] =
+    useState(false)
+  // store referrer id
+  useEffect(() => {
+    let referrerId = null
+    // Check for a referrer's vanity URL.
+    if (pageContext && pageContext.referrer) {
+      referrerId = pageContext.referrer.id
+    } else {
+      const paramRefId = parseInt(getUrlParameterValue('r'))
+      if (!isNaN(paramRefId)) {
+        referrerId = paramRefId
+      }
+    }
+    referrerId &&
+      localStorageMgr.setItem(
+        STORAGE_REFERRAL_DATA_REFERRING_CHANNEL,
+        referrerId
+      )
+  }, [])
+  // store user referral
+  useEffect(() => {
+    const userReferrerId = getUrlParameterValue('u')
+    if (userReferrerId !== null && userReferrerId !== undefined) {
+      localStorageMgr.setItem(
+        STORAGE_REFERRAL_DATA_REFERRING_USER,
+        userReferrerId
+      )
+    }
+  }, [])
+
+  // function to be used in install button
+  // eslint-disable-next-line no-unused-vars
+  const onBeforeInstall = () => {
+    localStorageMgr.setItem(STORAGE_NEW_USER_IS_TAB_V4_BETA, 'true')
+    localStorageMgr.setItem(STORAGE_NEW_USER_CAUSE_ID, causeId)
+  }
+  const absolutePageURL = getAbsoluteURL(location.pathname)
+  let headerImg
+  switch (causeId) {
+    case STORAGE_SEAS_CAUSE_ID:
+      headerImg = seasHeaderImg
+      break
+    case STORAGE_CATS_CAUSE_ID:
+      headerImg = catsHeaderImg
```

Comment:
it looks like the cats header image didn't actually export the cat from figma.  You'll want to re-add the cats Header image